css/hide_tooltips.css

```diff
@@ -1,3 +1,9 @@
+/* Hide tooltips for BS2 */
 .description {
   display: none;
 }
+
+/* Hide tooltips for BS5 */
+[id*='\-help'] {
+  display: none;
+}
```

Comment:
Some notes:

- This should go in the B5 theme SCSS files.
- How are they going to be shown based on the setting? This won't work with the existing B2 JS (which I believe is not even included in the B5 theme).
- Based on the discussion with @fiver-watson in Slack, we should only hide/show the help text for "standards-based guidance in the edit forms", not all input's help text.

And suggestions:

- Because they are no longer tooltips, instead of using JS to show/hide them, I'd add a class to the body based on the setting value so they are only hidden when the setting is set to not show tooltips. And add a new class to the help text that needs to be shown/hidden.
- I'd also change the setting name to something more accordingly with the current implementation, like "Show standards-based help text" instead of "Show Tooltips", but check with @fiver-watson first ;)

Reply:
hi @jraddaoui ! i looked into doing it only in BS5 but that would be a lot more work - which is fine but i want to first argue my approach XD

first off, these `*-help` ids are only used in the `arDominionB5Plugin` directory (checked by running `grep -ri --include='*.php' 'id.*\-help'`) and these changes doesn't affect how they work in BS2.

it is triggered by the [QubitDescription.class.php](https://github.com/artefactual/atom/blob/qa/2.x/lib/QubitDescription.class.php), so the setting will toggle these on/off, and only those for descriptions, not all `*-help` classes (so those in settings, csv, etc. are all excluded from this change). because of this, i had to use the `<?php include stylesheets(); ?>` in the `_layout_start_webpack.php` for this change to take affect.

if you still think this approach is not best practice and should be addressed only in `arDominionB5Plugin`, i can definitely try that too!

Reply:
I see, thanks @melaniekung!

I still think it should go in the B5 theme, it will be easy to forget this when the B2 themes are fully removed. Also, `<?php include stylesheets(); ?>` may be including other styles that we don't want in B5.